src/popup/components/AccountCardConsensus.vue

```diff
@@ -9,9 +9,7 @@
       <PendingIcon class="icon pending" />
       {{ $t('multisig.consensus') }}
       <span class="highlighted">
-        {{ currentAccount.confirmedBy.length }}/{{ currentAccount.confirmationsRequired }}
-        {{ $t('common.of') }}
-        {{ totalSignatures }}
+        {{ multisigAccount.consensusLabel }}
```

Comment:
we cannot use multisig Account.consensus Label bart found big problem with that. (this is not reloaded after changed languages)